From a4bc33b205fd9b1db862f1e45173dba57b0fa57f Mon Sep 17 00:00:00 2001
From: Ezequiel Garcia <ezequiel.garcia@imgtec.com>
Date: Sat, 25 Feb 2017 15:43:09 +0000
Subject: mtd: nand: Add JEDEC manufacturer ID for Gigadevice

This commit adds Gigadevice to the list of manufacturer ID and name strings.

(picked from http://lists.infradead.org/pipermail/linux-mtd/2014-December/056765.html)

Signed-off-by: Ezequiel Garcia <ezequiel.garcia@imgtec.com>
---
 drivers/mtd/nand/nand_ids.c | 1 +
 include/linux/mtd/rawnand.h    | 1 +
 2 files changed, 2 insertions(+)

--- a/drivers/mtd/nand/nand_ids.c
+++ b/drivers/mtd/nand/nand_ids.c
@@ -184,6 +184,7 @@ static const struct nand_manufacturer na
 	{NAND_MFR_SANDISK, "SanDisk"},
 	{NAND_MFR_INTEL, "Intel"},
 	{NAND_MFR_ATO, "ATO"},
+	{NAND_MFR_GIGADEVICE, "Gigadevice"},
 	{NAND_MFR_WINBOND, "Winbond"},
 };
 
--- a/include/linux/mtd/rawnand.h
+++ b/include/linux/mtd/rawnand.h
@@ -1014,6 +1014,7 @@ static inline void *nand_get_manufacture
 #define NAND_MFR_SANDISK	0x45
 #define NAND_MFR_INTEL		0x89
 #define NAND_MFR_ATO		0x9b
+#define NAND_MFR_GIGADEVICE	0xc8
 #define NAND_MFR_WINBOND	0xef
 
 
